distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Regex/Engine/Default.enso

```diff
@@ -132,9 +132,9 @@ type Default_Engine
             Java_Pattern.compile (unicode_regex.transform expression) options_bitmask
 
         internal_pattern = maybe_java_pattern.map_error case _ of
-            Polyglot_Error.Polyglot_Error_Data err ->
-                if err.is_a PatternSyntaxException . not then err else
-                    Syntax_Error_Data ("The regex could not be compiled: " + err.getMessage)
+            Polyglot_Error.Error (err : PatternSyntaxException) ->
+                Syntax_Error.Error ("The regex could not be compiled: " + err.getMessage)
+            Polyglot_Error.Error err -> err
```

Comment:
I guess @hubertp's comment from below applies here too, right?

Btw. @hubertp can you explain to us what this desugars to and why it should be avoided in this case?

Reply:
Removed this usage as well.